### ISSUE TITLE
ConfigCheck Dashboard Widget Should Alert User if MODX is Not Updated

### ISSUE THREAD

**Issue by muzzwood** [modxcms/revolution#14020]
### Summary
The ConfigCheck dashboard widget should check if the installed MODX is the latest version.

### Step to reproduce
View the dashboard on login to manager

### Observed behavior
There is currently no alert that an older insecure version of MODX may be installed.

### Expected behavior
There should be something very visible to alert a user to the fact their install is out of date.

### Environment
All versions of MODX.

**Comment by JoshuaLuckers** [comment]
I MODX 3 there is a "updates" widget. I'm not sure if it's functional yet. 

![updates widget](https://user-images.githubusercontent.com/1268085/43380798-c2b62f0c-93d2-11e8-829d-fe6236bea233.png)

There are also several related issues about better informing and making it easier to upgrade: #14007, #13715 (and probably more).

**Comment by muzzwood** [comment]
Good point. Although I think this should also be something that targets 2.7 as there would be security releases for it even once MODX 3 is in the wild. 

Is there an endpoint to check for the current version of MODX yet? 
I know @bobray was asking about it for UpgradeMODX being incorporated into the core too.

**Comment by alroniks** [comment]
> Is there an endpoint to check for the current version of MODX yet?
not yet

And, I am going to work in upgrading feature for MODX (I guess as it shouldn't contain any BC, so can be applied into 2.7 too) in collaboration with BobRay in the next month (I don't want to give a more short promise, because I will be a support guy during the August at the main work).

**Comment by AlexUnicom** [comment]
Hello guys! Can somebody help me to resolve it? If I upgrade my MODX Revolution 2.2.14-pl (traditional) to 2.6.5-pl version the catalogs on site do not work. And if update FormIt extra then an error appears on screen only.

**Comment by muzzwood** [comment]
@AlexUnicom Sorry, but this is not the appropriate place. Try asking your question at https://forums.modx.com/
